core/events/events_procedure_create.ts

```diff
@@ -4,13 +4,36 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import type {IProcedureModel} from '../interfaces/i_procedure_model.js';
+import {ObservableProcedureModel} from '../procedures.js';
 import * as registry from '../registry.js';
+import type {Workspace} from '../workspace.js';
 
 import {ProcedureBase} from './events_procedure_base.js';
 import * as eventUtils from './utils.js';
 
 
-export class ProcedureCreate extends ProcedureBase {}
+export class ProcedureCreate extends ProcedureBase {
+  type = eventUtils.PROCEDURE_CREATE;
+
+  constructor(workspace: Workspace, model: IProcedureModel) {
```

Comment:
Isn't this change of signature (constructor now has required arguments) a breaking change?

Reply:
These events/classes haven't been published, so no breaking change (here and below).

Reply:
Whew!